BUILD

```diff
@@ -1,36 +1,23 @@
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 
-exports_files(
-    [
-        "versions/master/docs/bazel-user-manual.html",
-        "command-line-reference-prefix.html",
-        "command-line-reference-suffix.html",
-    ],
-)
-
-DOT_GRAPH_HTML_FILES = [
-    "versions/master/docs/query.html",
-    "versions/master/docs/build-ref.html",
-]
-
-filegroup(
-    name = "srcs",
-    srcs = glob(["**"]),
-    visibility = ["//:__pkg__"],
-)
-
 filegroup(
     name = "jekyll-srcs",
     srcs = glob(
         ["**/*"],
         exclude = [
+            ".git/**",
+            "bazel-*/**",
             "BUILD",
+            "WORKSPACE",
             "jekyll-tree.sh",
+            "scripts/**",
             "*.swp",
-            "command-line-reference-prefix.html",
-            "command-line-reference-suffix.html",
             "site/README.md",
-        ] + DOT_GRAPH_HTML_FILES,
+            "LICENSE",
+            "CONTRIBUTING",
+            "production/**",
+            "README.md",
+        ],
     ),
```

Comment:
Tried renaming back to bazel-release and using the following `filegroup` rule:

```python
filegroup(
    name = "jekyll-srcs",
    srcs = [
        "bazel-release.pub.gpg",
    ] + glob(
        ["**/*"],
        exclude = [
            ".git/**",
            "bazel-*/**",
            "BUILD",
            "WORKSPACE",
            "jekyll-tree.sh",
            "scripts/**",
            "*.swp",
            "site/README.md",
            "LICENSE",
            "CONTRIBUTING",
            "production/**",
            "README.md",
        ],
    ),
)
```

But still getting the same error seen in bazelbuild/bazel#2909:

```
❯❯ scripts/serve-docs.sh
expr: syntax error
INFO: Analysed target //:jekyll-tree.tar (9 packages loaded).
INFO: Found 1 target...
ERROR: missing input file '//:bazel-release.pub.gpg'
ERROR: /Volumes/Ocean/Projects/bazelbuild/bazel-website/BUILD:26:1: //:jekyll-files: missing input file '//:bazel-release.pub.gpg'
Target //:jekyll-tree.tar failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /Volumes/Ocean/Projects/bazelbuild/bazel-website/BUILD:26:1 1 input file(s) do not exist
INFO: Elapsed time: 3.943s, Critical Path: 0.01s
FAILED: Build did NOT complete successfully
```